specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-02-01-preview/desktopvirtualization.json

```diff
@@ -3865,14 +3865,70 @@
             "Shutdown",
             "Disconnected",
             "Upgrading",
-            "UpgradeFailed"
+            "UpgradeFailed",
+            "NoHeartbeat",
+            "NotJoinedToDomain",
+            "DomainTrustRelationshipLost",
+            "SxSStackListenerNotReady",
+            "FSLogixNotHealthy",
+            "NeedsAssistance"
           ],
           "x-ms-enum": {
             "name": "Status",
-            "modelAsString": true
-          },
-          "type": "string",
-          "description": "Status for a SessionHost. Available - Session Host has passed all the health checks and is available to handle connections. Unavailable - Session Host is either turned off or has failed critical health checks which is causing service not to be able to route connections to this session host. Note this replaces previous 'NoHeartBeat' status. Shutdown - Session Host is shutdown - RD Agent reported session host to be stopped or deallocated. Disconnected - The Session Host is unavailable because it is currently disconnected. Upgrading - Session Host is unavailable because currently an upgrade of RDAgent/side-by-side stack is in progress. Note: this state will be removed once the upgrade completes and the host is able to accept connections. UpgradeFailed - Session Host is unavailable because the critical component upgrade (agent, side-by-side stack, etc.) failed."
+            "modelAsString": false,
```

Comment:
Are you sure you want modelAsString=false? This could mean whenever you add enum, you need a new api-version.

Reply:
I want to add the documentation and was following an example. Ok i will turn on the modelAsString!